packages/datadog-plugin-pg/src/index.js

```diff
@@ -27,6 +27,10 @@ class PGPlugin extends DatabasePlugin {
       }
     })
 
+    if (stream) {
+      span.setTag('db.stream', true)
```

Comment:
```suggestion
      span.setTag('db.stream', 1)
```

I am not sure, but I guess we could just directly set it to one in case we later on translate that anyway?

Reply:
done.